lib/containers/containerd_crictl.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2020-2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: engine subclass for containerd with crictl specific implementations
+# Maintainer: qac team <qa-c@suse.de>
+
+package containers::containerd_crictl;
+use Mojo::Base 'containers::engine';
+use testapi;
+use containers::common 'install_containerd_when_needed';
+use containers::utils 'registry_url';
+has runtime => 'crictl';
+
+sub init {
+    install_containerd_when_needed();
+
+    # The crictl validation test suite is a plug-in required for this test and needs to be installed from an external source.
+    my $url = 'https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.23.0/crictl-v1.23.0-linux-amd64.tar.gz';
```

Comment:
nitpicking: can we make the package URL dynamic?

`CONTAINERS_CRICTL_VERSION=v1.23.0`

```suggestion
    my $url = "https://github.com/kubernetes-sigs/cri-tools/releases/download/$CONTAINERS_CRICTL_VERSION/crictl-$CONTAINERS_CRICTL_VERSION-linux-amd64.tar.gz";
```